Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+@Library('xmos_jenkins_shared_library@v0.27.0') _
+
+pipeline {
+    agent none
+
+    options {
+        disableConcurrentBuilds()
+        skipDefaultCheckout()
+        timestamps()
+        buildDiscarder(xmosDiscardBuildSettings(onlyArtifacts=false))
+    }
+    parameters {
+        string(
+            name: 'TOOLS_VERSION',
+            defaultValue: '15.2.1',
+            description: 'The XTC tools version'
+        )
+    }
+
+    stages {
+        stage("Do Jenkins") {
+            agent {
+                label 'linux&&64'
+            }
+            stages {
+                stage ("Do Jenkins") {
+                    steps {
+                        withTools(params.TOOLS_VERSION){ 
+                            sh "mkdir -p my-sandbox/lib_locks"
```

Comment:
The dir() command creates directories as required, so mkdir is not needed.

Reply:
done